# Message to control high level actions of the Ego-Vehicle


# lane_change : Change lane (0 keep lane; 1 lane change right; 2 lane change left)
# accel : Acceleration value (0 remain; 1 accelerate; -1 brake; -2 hard brake; -100 Use SUMO controlled speed)

int16 lane_change
int16 accel